tensorflow/lite/python/convert.py

```diff
@@ -94,6 +94,12 @@ class OpsSet(enum.Enum):
   # quantized implementations.
   TFLITE_BUILTINS_INT8 = "TFLITE_BUILTINS_INT8"
 
+  # Convert model using only TensorFlow Lite operations with quantized int8 weights
+  # and int16 activations.
+  # Specifying this will throw an error for operations that do not yet have
+  # quantized implementations.
+  EXPERIMENTAL_TFLITE_BUILTINS_ACTIVATIONS_INT16_WEIGHTS_INT8 = "EXPERIMENTAL_TFLITE_BUILTINS_ACTIVATIONS_INT16_WEIGHTS_INT8"
```

Comment:
Let's add the following:
```
# WARNING: These ops are currently experimental and have not yet been finalized.
# They are only compatible with CPU execution, and have not been optimized for production.
```

Reply:
Also, as this will go through API review, I think it would be good to have a sentence or two about when to use this flag vs TFLITE_BUILTINS_INT8.